pip_compile/Tiltfile

```diff
@@ -0,0 +1,39 @@
+load("ext://uibutton", "cmd_button")
+
+
+SCRIPT = os.path.join(os.getcwd(), "scripts", "controller.py")
+DEFAULT_COMPILE_ARGS = ["--allow-unsafe", "--generate-hashes"]
+
+
+def pip_compile_button(
+    image,
+    target = "",
+    reqs_path = ".",
+    compile_args = DEFAULT_COMPILE_ARGS,
+):
+    """Add `pip-compile` button to all Tilt resources using `image`.
```

Comment:
My editor wants the function arguments to be documented in the docstring, and I agree.

<img width="643" alt="image" src="https://github.com/DirectEmployers/tilt-extensions/assets/5957867/41e5e7ad-53b1-461c-aeb1-0ff6717fc91a">

Reply:
This can be done :)